package.json

```diff
@@ -12,7 +12,7 @@
     "build:icosium": "cd icosium && yarn build",
     "test:icosium": "cd icosium && yarn test",
     "eject:icosium": "cd icosium && yarn eject",
-    "dev:hoggar": "cd hoggar && yarn dev",
+    "dev:hoggar": "cd hoggar &&  yarn dev",
```

Comment:
- add prisma client as dependecy
- scripts to generate and migrate on root package.json
- add steps in DOC